breachCheck.c

```diff
@@ -0,0 +1,32 @@
+#include "config.h"
+#include "breachCheck.h"
+
+BreachType inferBreach(double value, double lowerLimit, double upperLimit) 
+{
+  if(value < lowerLimit) 
+  {
+    return TOO_LOW;
+  }
+  if(value > upperLimit) 
+  {
+    return TOO_HIGH;
+  }
+  return NORMAL;
+}
+
+BreachType classifyTemperatureBreach(CoolingType coolingType, double temperatureInC) 
+{
+   int lowerLimit = 0;
+   int upperLimit = 0;
+   for ( int CoolingInfoTableIndex = 0; CoolingInfoTableIndex < MAXIMUM_COOLING_TYPE;CoolingInfoTableIndex++)
```

Comment:
Cooling tyoe code can be refactored to a different method to maintain readability and reduce complexity.